specification/columns/commitmentconsumedquantity.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Consumed Quantity
```

Comment:
Seen as used Commitment discount quantity shows up in ConsumedQuantity, do we really need CommitmentConsumedQuantity? Other columns, how about CommitmentTotalQuantity becomes CommitmentDiscountQuantity - this would be the amount of commitment ordered for example 15 units of SQL DW, CommitmentConsumedUnit becomes CommitmentDiscountUnit. Use descriptions to allow practitioners/providers understand the required data in the columns instead of prescribing the data in the column name.

Reply:
Were you able to look at the sample data for the commitment pr?  In 2 of 3 cases (at least for AWS, Azure), the consumed quantity/units of the resource and the commitment consumed quantity/units of the commitment are different values?

```
- RI (zonal) ✅ 
    - Consumed Unit -> Hours
    - Commitment Consumed Unit -> Hours
- RI (regional) ❌ 
    - Consumed Unit -> Hours
    - Commitment Consumed Unit -> Normalized Hours
- Savings Plans ❌ 
    - Consumed Unit -> Hours
    - Commitment Consumed Unit -> <Currency>
```

Reply:
`CommitmentConsumedUnit` complements both `CommitmentConsumedQuantity` and `CommitmentTotalQuantity` therefor I suggest we use the term `CommitmentUnit`

Reply:
<img width="757" alt="image" src="https://github.com/user-attachments/assets/c1be1dee-1ddb-4f2d-9075-210d07b46210">